notebooks/0.freature_selection/0.feature_selection.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
the folder name here is still misspelled (`0.freature_selection` -> `0.feature_selection`)

(This attention to detail is important for a data scientist! 😄 )